UnityProject/Assets/IngameDebugConsole/Scripts/DebugLogManager.cs

```diff
@@ -588,6 +588,22 @@ public void FilterErrorButtonPressed()
 			FilterLogs();
 		}
 
+		/// <summary>
+		/// Creates a local profile
+		/// </summary>
+		public void LocalProfileButtonPressed()
+		{
+			if (Debug.isDebugBuild)
```

Comment:
this won't work for the clients we distribute then

Reply:
yeah but thats just how it works

Reply:
I know,. Still nice to have